opencog/cython/opencog/spatial.pyx

```diff
@@ -105,10 +105,9 @@ cdef class EntityRecorder:
     def __cinit__(self):
         pass
 
-    def __init__(self, long addr, AtomSpace atomspace):
+    def __init__(self, long addr):
         self.c_entity_recorder = <cEntityRecorder*> PyLong_AsVoidPtr(addr)
-        self.atomspace = atomspace
-
+ 
```

Comment:
are you sure?  This now differs from how `__init__` works for OctomapOCTree up above. Also, it looks like now, self.atomspace is never initialized; yet it is used by more-or-less every method in this class ...

Atoms, once inserted into some atomspace, always know what atomspace they are in.  Only fresh atoms, that have never been inserted anywhere, don't know where they belong., so they're left floating about.   Maybe the atomspace argument isn't needed for most of these functions? Maybe passing an invalid atomspace is OK ???